### ISSUE TITLE
Music Plays fast

### ISSUE THREAD

**Issue by a7mag3ddon** [lantus/cannonball-nx#2]
The music seems to play faster than the arcade, I'm not sure If the whole emulator is running a little bit fast or just the music.

**Comment by lantus** [comment]
yep i noticed that too, i will check into it. if you look at the countdown clock its moving slightly faster than normal

**Comment by a7mag3ddon** [comment]
Perhaps it's possible to lock the FPS to 50? That may help.

**Comment by Dizzy611** [comment]
I kind of like the too-fast gameplay as an enhancement over the original, as (at least for me) it's still quite smooth. Maybe make it something you can toggle?

**Comment by a7mag3ddon** [comment]
Music runs fast because the emulator is running fast. Needs fps locking.
